odoo-shippable/scripts/build-image.sh

```diff
@@ -15,6 +15,9 @@ ARCH="$( dpkg --print-architecture )"
 GITCORE_PPA_REPO="deb http://ppa.launchpad.net/git-core/ppa/ubuntu trusty main"
 GITCORE_PPA_KEY="http://keyserver.ubuntu.com:11371/pks/lookup?op=get&search=0xA1715D88E1DF1F24"
 
+# ppa sources
+PPA_SOURCES="fkrull/deadsnakes"
+
```

Comment:
@KarenKawaii Maybe you should change this for a descriptive name like [here](https://github.com/Vauxoo/docker-odoo-image/blob/master/odoo-shippable/scripts/build-image.sh#L15) so we can know what is this actually for